platform/machine/brightbox/flight.go

```diff
@@ -0,0 +1,79 @@
+// Copyright The Mantle Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package brightbox
+
+import (
+	"fmt"
+
+	"github.com/coreos/pkg/capnslog"
+	ctplatform "github.com/flatcar/container-linux-config-transpiler/config/platform"
+
+	"github.com/flatcar/mantle/platform"
+	"github.com/flatcar/mantle/platform/api/brightbox"
+)
+
+const (
+	Platform platform.Name = "brightbox"
+)
+
+var (
+	plog = capnslog.NewPackageLogger("github.com/flatcar/mantle", "platform/machine/brightbox")
+)
+
+type flight struct {
+	*platform.BaseFlight
+	api      *brightbox.API
+	cloudIPs chan string
+}
+
+func NewFlight(opts *brightbox.Options) (platform.Flight, error) {
+	api, err := brightbox.New(opts)
+	if err != nil {
+		return nil, fmt.Errorf("creating brightbox API client: %w", err)
+	}
+
+	base, err := platform.NewBaseFlight(opts.Options, Platform, ctplatform.OpenStackMetadata)
+	if err != nil {
+		return nil, fmt.Errorf("creating base flight: %w", err)
+	}
+
+	bf := &flight{
+		BaseFlight: base,
+		api:        api,
+		// Current CloudIPs limit is 5.
+		cloudIPs: make(chan string, 5),
```

Comment:
This should be a cli flag

Reply:
Or: The use of "select default" means it won't wait anyway, I guess it would work to do 
```suggestion
		cloudIPs: make(chan string, 999),
```
because the value has no meaning.